### ISSUE TITLE
story: getting job status

### ISSUE THREAD

**Issue by yashipro13** [gopay-bootcamp/octavius#8]
#### Description
As a client, he should have an accessibility to view status of the job in system by executing command `get-stream <id>` , where id is unique id associated with the job.

#### Present Behaviour

No such behaviour

#### Expected Behaviour

The client should be able to execute the command `get-stream <id>` and get the stream status of job in response.

#### Screenshots

No screenshots applicable
